routers/routers.js

```diff
@@ -0,0 +1,7 @@
+const router = require('express-promise-router')();
+const {Signup, logIn} = require('../controllers/UserController');
+
+router.route("/signup").post(Signup);
```

Comment:
@isiagi  Good work looking at the project l think you are supposed to use versioning and also the route is supposed to be defined as `api/v1/auth/signup` instead of signup Please refer to the project guideline.